libraries/AP_UAVCAN/AP_UAVCAN.cpp

```diff
@@ -511,9 +736,26 @@ bool AP_UAVCAN::try_init(void)
                     rgb_led[_uavcan_i] = new uavcan::Publisher<uavcan::equipment::indication::LightsCommand>(*node);
                     rgb_led[_uavcan_i]->setTxTimeout(uavcan::MonotonicDuration::fromMSec(20));
                     rgb_led[_uavcan_i]->setPriority(uavcan::TransferPriority::OneHigherThanLowest);
-
                     _led_conf.devices_count = 0;
 
+                    if(_broadcast_bm & AP_UAVCAN_BROADCAST_POSITION_FIX) {
```

Comment:
I think something was lost in the rebase. No param sets _broadcast_bm so these subscribers will never init. We should always init them and just control the rates in the update/cyclic. Keeping like this requires a reboot if you turn it on after the boot which is not desirable - It should just start working.

Reply:
It was not lost in rebase, but a bug that is a left over after changing the rate logic.